massa-models/src/block.rs

```diff
@@ -583,28 +583,27 @@ mod test {
                 endorsements: vec![
                     Wrapped::new_wrapped(
                         Endorsement {
-                            sender_public_key: public_key,
                             slot: Slot::new(1, 1),
                             index: 1,
                             endorsed_block: BlockId(Hash::compute_from("blk1".as_bytes())),
                         },
+                        EndorsementSerializer::new(),
                         &private_key,
                     )
-                    .unwrap()
-                    .1,
+                    .unwrap(),
                     Wrapped::new_wrapped(
                         Endorsement {
-                            sender_public_key: public_key,
                             slot: Slot::new(4, 0),
                             index: 3,
                             endorsed_block: BlockId(Hash::compute_from("blk2".as_bytes())),
                         },
+                        EndorsementSerializer::new(),
```

Comment:
could we share the EndorsementSerializer created above instead of creating a new one ?